gulpfile.coffee

```diff
@@ -42,9 +39,18 @@ gulp.task 'lint', ->
 		.pipe(coffeelint.reporter())
 
 gulp.task 'build', (callback) ->
-	runSequence([
-		'lint'
-	], [ 'coffee' ], callback)
+	runSequence('lint', 'build-node', 'build-browser', callback)
+
+gulp.task 'build-node', ->
+	gulp.src(OPTIONS.files.app)
+		.pipe(coffee(header: true)).on('error', gutil.log)
+		.pipe(gulp.dest(OPTIONS.directories.build))
+
+gulp.task 'build-browser', ['build-node'], ->
+	browserify(entries: OPTIONS.files.browserEntry, standalone: 'resin-sdk')
+		.bundle()
+		.pipe(source('resin-browser.js'))
```

Comment:
Am I right that currently it does include all and every dependency?
If so I have a list of deps that are definitely not used in the browser to exclude:

```
 "browserify src/index.js -o build/bundle.js --exclude fs --exclude path --exclude resin-settings-client --exclude node-localstorage --exclude rindle --exclude zlib --exclude stream --exclude progress-stream --exclude request"
```

The last one (`request`) used to be needed because of the `resin-register-device` and can probably be dropped now.

The expected minified and gzipped bundle size is around 70Kb. At leats that's what I got locally with the above command.